shared/reducers/chat2.js

```diff
@@ -81,6 +81,9 @@ const metaMapReducer = (metaMap, action) => {
               conversationIDKey,
               Constants.makeConversationMeta({
                 conversationIDKey,
```

Comment:
we should just consume the entire InboxUIItemError and not just the membershipType

Reply:
added a helper to eat the InboxItem, PTAL